packages/react/ds/src/heading/heading.stories.tsx

```diff
@@ -14,3 +14,16 @@ export const Default: Story = {
     children: 'Heading',
   },
 };
+
+// @ts-expect-error
+export const WithSize: Story = {
+  render: () => (
+    <div className="flex flex-row gap-2">
+      {Object.values(HeadingSize).map((size) => (
+        <Heading key={size} size={size}>
+          Heading {size}
+        </Heading>
+      ))}
+    </div>
+  ),
```

Comment:
in the Jinja/Nunjucks library, we have a story for each size variant. Keeping in mind that we looking to achieve consistency between the two libraries when it comes to storybook, are we going with one story that renders all sizes like the one above or do we want to have a story for each size ? WDYT ?

Reply:
Can we achieve the same also with the macro?

Reply:
I don't believe that we can. Since we have a `.ts` extension for the macros we cannot use the tsx features. Of course we can try to change the extension from `.ts` to `.tsx` but we might get some unexpected results if we do that since they are not React components

Reply:
the render function in Jinja should return a html string, we have an example in html/ds/button/button.stories.ts (AllVariants) story.

Reply:
Resolving this since we removed enums in favour of non-breaking change with tailwind-variants